site/docs/v1/tech/apis/_user-registration-combined-request-body.adoc

```diff
@@ -109,7 +109,45 @@ The User's preferred timezone. The string must be in an https://www.iana.org/tim
 [quote]
 `America/Denver` or `US/Mountain`
 
-[field]#users.twoFactorDelivery# [type]#[String]# [optional]#Optional# [default]#defaults to `None`#::
+[field]#user.twoFactor.methods``[x]``.authenticator.algorithm# [type]#[TOTPAlgorithm]# [optional]#Optional#::
+The algorithm used by the TOTP authenticator. With the current implementation, this will always be `HmacSHA1`.
+
+[field]#user.twoFactor.methods``[x]``.authenticator.codeLength# [type]#[Integer]# [optional]#Optional#::
+The length of code generated by the TOTP. With the current implementation, this will always be `6`.
+
+[field]#user.twoFactor.methods``[x]``.authenticator.timeStep# [type]#[Integer]# [optional]#Optional#::
+The time-step size in seconds. With the current implementation, this will always be 30.
+
+[field]#user.twoFactor.methods``[x]``.email# [type]#[String]# [optional]#Optional#::
+The value of the email address for this method. Only present if [field]#user.twoFactor.methods``[x]``.method# is `email`.
+
+[field]#user.twoFactor.methods``[x]``.id# [type]#[UUID]# [optional]#Optional#::
```

Comment:
Not a UUID (as per other `methods` comments) but a string.

Reply:
Also can you even provide the id?

Reply:
Good catch - you should not be passing an Id here.